docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3.8"
+
+x-common-variables:
+  read-only-workdir: &read-only-workdir
+    type: bind
+    source: .
+    target: /workdir
+    read_only: true
+  read-write-workdir: &read-write-workdir
+    type: bind
+    source: .
+    target: /workdir
+    read_only: false
+  read-only-plugin: &read-only-plugin
+    # Buildkite containers assume you mount into /plugin
+    type: bind
+    source: .
+    target: /plugin
+    read_only: true
+
+services:
+  hcl-formatter:
+    image: docker.cloudsmith.io/grapl/releases/hcl-formatter:latest
+    command: format
+    volumes:
+      - *read-write-workdir
+
+  hcl-linter:
+    image: docker.cloudsmith.io/grapl/releases/hcl-formatter:latest
+    command: lint
+    volumes:
+      - *read-only-workdir
+
+  hadolint:
```

Comment:
Mentioned later on as well, but pants has support for hadolint. Given the number of hadolint warnings in the Grapl repo when I tried it, it may make sense to enable this for the smaller repos first